web_src/js/modules/stores.js

```diff
@@ -2,4 +2,9 @@ import {reactive} from 'vue';
 
 export const DiffTreeStore = reactive({
   selectedItem: '',
+  isIncomplete: false,
+  diffEnd: 0,
+  link: '',
+  isLoadingNewData: false,
+  files: []
```

Comment:
Can `files` be made a `Set`? There can not be duplicates in the file paths. Also I think `files` is not a good name, maybe `viewedFiles`?

Reply:
It's not only a tracker of the viewed files - it's the list of all files shown in the diff.

Reply:
Ok, still could be made a `Set` then, to make existance checks `O(1)`.